asr_kaldi/README.md

```diff
@@ -4,7 +4,7 @@
 
 ## Pre-requisites
 - 0MQ
-- OpenVINO&trade; 2021.3
+- OpenVINO&trade; 2020.4
```

Comment:
2021.3
https://github.com/intel-innersource/applications.ai.conversational-ai.chatbot/blob/master/dockerfiles/kaldi_asr.dockerfile

Reply:
fixed in cabf1272c5240e39836103f5147554ca8ea86acb